tests/test_model_builder.py

```diff
@@ -79,62 +77,62 @@ async def test_build_model(self, pgpool, postgresql):
             assert TEST_MANUAL_AGORA_ENTRY not in model
 
     async def test_build_model_with_manual_entry(self, pgpool, postgresql):
-        # Creates a temp file and dir to simulate manual entries
+        subgraph = "Qmadj8x9km1YEyKmRnJ6EkC2zpJZFCfTyTZpuqC3j6e1QH"
         file_type = ".agora"
-        os.makedirs(user_config_dir("autoagora"), exist_ok=True)
-        temp_file = tempfile.NamedTemporaryFile(
-            suffix=file_type, dir=user_config_dir("autoagora"), delete=False
-        )
-        byte_entry = bytes(TEST_MANUAL_AGORA_ENTRY, "ascii")
-        temp_file.write(byte_entry)
-        temp_file.close()
-        with mock.patch(
-            "autoagora.logs_db.LogsDB.get_most_frequent_queries"
-        ) as mock_get_mfq:
-            mock_get_mfq.return_value = [
-                LogsDB.QueryStats(
-                    query=TEST_QUERY_1,
-                    count=100,
-                    min_time=1,
-                    max_time=60,
-                    avg_time=1.2,
-                    stddev_time=0.5,
-                ),
-                LogsDB.QueryStats(
-                    query=TEST_QUERY_2,
-                    count=10,
-                    min_time=3,
-                    max_time=20,
-                    avg_time=13.2,
-                    stddev_time=0.7,
-                ),
-            ]
-            init_config(
-                [
-                    "--indexer-agent-mgmt-endpoint",
-                    "http://nowhere",
-                    "--postgres-host",
-                    postgresql.info.host,
-                    "--postgres-username",
-                    postgresql.info.user,
-                    "--postgres-password",
-                    postgresql.info.password,
-                    "--postgres-port",
-                    str(postgresql.info.port),
-                    "--postgres-database",
-                    postgresql.info.dbname,
-                    "--indexer-service-metrics-endpoint",
-                    "http://indexer-service.default.svc.cluster.local:7300/metrics",
+        # Creates a temp dir to simulate manual entries
+        with tempfile.TemporaryDirectory() as temp_dir:
+            # Create a file inside temp dir which will be deleted once out of context
+            temp_file = open(os.path.join(temp_dir, subgraph + file_type), "w")
+            print(temp_file.name)
+            temp_file.write(TEST_MANUAL_AGORA_ENTRY)
+            temp_file.close()
```

Comment:
Can we use a context manager here?

Reply:
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory
No need since it deletes all its items after finishing context

Reply:
I was meaning the file creation
```suggestion
            with open(os.path.join(temp_dir, subgraph + file_type), "w") as temp_file:
                temp_file.write(TEST_MANUAL_AGORA_ENTRY)
            print(temp_file.name)
```